packages/realm_dart/lib/src/realm_object.dart

```diff
@@ -771,6 +784,22 @@ class DynamicRealmObject {
     return RealmObjectBase.get<T>(_obj, name) as RealmList<T>;
   }
 
+  /// Gets a set by the property name. If a generic type is specified, the property
+  /// type will be validated against the type. Otherwise, a `Set<Object>` will be
+  /// returned.
+  RealmSet<T> getSet<T extends Object?>(String name) {
+    _validatePropertyType<T>(name, RealmCollectionType.set);
+    return RealmObjectBase.get<T>(_obj, name) as RealmSet<T>;
+  }
+
+  /// Gets a map by the property name. If a generic type is specified, the property
+  /// type will be validated against the type. Otherwise, a `Map<String, Object>` will be
```

Comment:
```suggestion
  /// type will be validated against the type. Otherwise, a `RealmMap<Object?>` will be
```
Isn't it important to state that it is not just a map?